### ISSUE TITLE
A uniform input for all tools in each screening workflow

### ISSUE THREAD

**Issue by m3hdad** [nf-core/funcscan#385]
### Description of feature

Currently different analysis software in ARG screening workflow use different input sequences.
For instance, AMRFinderPlus and RGI use nucleotide sequence as input whereas deeparg uses predicted protein sequences from the annotation step in the beginning of the pipeline.

Since all software accept both nucleotide and protein sequences, it would be really helpful to have a uniform input for all software which allows post-processing and comparison between predictions by different software based on sequence accession number.

https://nfcore.slack.com/archives/C02K5GX2W93/p1717515107461539

**Comment by jfy133** [comment]
Yes this is a fair point. A little more context, from the one implied by Anan: essentially we want to try and use input that has already been annotated by the pipeline to ensure consistency and efficiency, rather than letting each tool do their own annotation each with a different tool.

So really I would almost want to go in the other direction (I think what the other issue Anan is pointing to ), use amino acid sequence (where possible). That said, we could theoretically make it user choice, but this would require a lot more more channel/conditional code to be added...
